inspirehep/base/templates/format/record/Inspire_Default_HTML_detailed.tpl

```diff
@@ -28,49 +29,59 @@
       {{ record_collection_heading() }}
     </div>
     <div id="record-title">
-      {{ record_title() }}
+      {{ render_record_title() }}
     </div>
     <div id="record-authors">
       {{ render_record_authors(is_brief=false, number_of_displayed_authors=25) }}
     </div>
+    <div id="record-year">
```

Comment:
<img width="173" alt="screen shot 2015-12-14 at 14 39 31" src="https://cloud.githubusercontent.com/assets/7301890/11782301/8db8c7fc-a270-11e5-9c3d-eeb08373c089.png">

I guess having it here it the best.